Documentation/helm-operator.md

```diff
@@ -101,8 +101,8 @@ The following tables lists the configurable parameters of the rook-operator char
 | `discover.podLabels`               | Labels to add to the discover pods.                                                                                            | <none>                                                    |
 | `csi.enableRbdDriver`              | Enable Ceph CSI RBD driver.                                                                                                    | `true`                                                    |
 | `csi.enableCephfsDriver`           | Enable Ceph CSI CephFS driver.                                                                                                 | `true`                                                    |
-| `csi.enableCephfsSnapshotter`      | Enable Snapshotter in CephFS provisioner pod.                                                                                  | `true`                                                    |
-| `csi.enableRBDSnapshotter`         | Enable Snapshotter in RBD provisioner pod.                                                                                     | `true`                                                    |
+| `csi.enableCephfsSnapshotter`      | Enable Snapshotter in CephFS provisioner pod.                                                                                  | if K8s >= v1.17 `true` else `false`                                       |
```

Comment:
I'd suggest we just keep this helm doc simple with the default of `true`. If they don't see it enabled, they can look at the CSI help topic for the details.